src/Sorting/ArrayFilter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\SwissKnife\Sorting;
+
+use Webmozart\Assert\Assert;
+
+final class ArrayFilter
+{
+    /**
+     * @param array<mixed[]> $items
+     * @return array<mixed[]>
+     */
+    public static function filterOnlyAtLeast2(array $items): array
+    {
+        Assert::allIsArray($items);
+
+        return array_filter($items, function (array $item): bool {
+            $nonEmptyValues = array_filter($item);
+
+            // +1 for the package name
+            return count($nonEmptyValues) >= 3;
```

Comment:
Should it be `>=2` for at least 2 based ob method name?

btw, you may looking for my ArrayLookup Finder::rows(), should be something like this that cover it :)

```php
use ArrayLookup\Finder;

$filter = fn (array $item): bool => count(array_filter($item)) >= 3;

return Finder::rows($items, $filter);
```

The tool allow preserve key if needed, see doc https://github.com/samsonasik/ArrayLookup?tab=readme-ov-file#c-finder

Reply:
The package name is here too :) e.g. this should be skipped:

```bash
[package, version, null]
```

The less deps the better.

Reply:
just a bit confused about method name :)

anyway, you may can try my `ArrayLookup\Finder::rows()` for this with simpler syntax like my suggestion above

I can create separate PR if you want so this custom logic is not needed

Reply:
oh, I see what you mean about deps, as dependency :)